app/livechat/server/api/v1/message.js

```diff
@@ -95,7 +96,7 @@ API.v1.addRoute('livechat/message/:_id', {
 				throw new Meteor.Error('invalid-message');
 			}
 
-			return API.v1.success({ message });
+			return API.v1.success({ message: FileUpload.addJWTToFileUrlIfNecessary(message) });
```

Comment:
As I commented before, IMO it would be better having a specific helper method to handle this process, and inside that method, I would call the `addJWTToFileUrlIfNecessary` method.
In that case, we could rename the `addJWTToFileUrlIfNecessary` to `addJWTToFileUrl`.